components/engine/src/util.rs

```diff
@@ -80,6 +80,7 @@ pub fn delete_all_in_range_cf(
                 break;
             }
         }
+        it.status()?;
```

Comment:
It seems to me `SnapContext::cleanup_range` only log error after getting error from `delete_all_in_range_cf`. Is it used for cleanup range after migrate region off the node? If so, how can we guarantee when the region migrate back, previous data had cleaned up?

Reply:
We must guarantee that the range has been totally cleanup before apply a snapshot.

Reply:
@Connor1996 You should check the result of cleanup_range at here https://github.com/tikv/tikv/blob/master/src/raftstore/store/worker/region.rs#L292.

Reply:
My fault, https://github.com/tikv/tikv/blob/master/src/raftstore/store/worker/region.rs#L293 has done this job.

Reply:
@zhangjinpeng1987 Is `cleanup_overlap_ranges` and `engine_util::delete_all_in_range` redundant to each other? Since `cleanup_overlap_ranges` effectively also call `delete_all_in_range`. Do I miss something?

Reply:
@Connor1996 If https://github.com/tikv/tikv/blob/master/src/raftstore/store/worker/region.rs#L292 and https://github.com/tikv/tikv/blob/master/src/raftstore/store/worker/region.rs#L293 are redundant, add a TODO there for combining them?

Reply:
seems it is a double check to make sure the range is clean? /cc @hicqu